### ISSUE TITLE
Docs still ref configmap style of configuration

### ISSUE THREAD

**Issue by haf** [zalando/postgres-operator#1267]
https://github.com/zalando/postgres-operator/blob/master/docs/reference/operator_parameters.md says the OperatorConfiguration is the new way of configuring the operator, yet the quickstart https://postgres-operator.readthedocs.io/en/latest/ references the config map when installing

**Comment by FxKu** [comment]
Yes, this is confusing, I admit. We aren't very consequent with this one as we still use the ConfigMap configuration internally. I think, when we will rewrite the messy go code responsible for configuring the operator we could finally really deprecate the ConfigMap way as CRD has advantages with nested structures and validation. It's on our long to do list for the next year(s).

**Comment by balibebas** [comment]
Quick Start docs should be updated to reference the "preferred" way and the current way. It should show both since the new way is intended to deprecate the old way as specified in the docs.
